Islandora-Image-Segmentation/Islandora-Image-Segmentation-Solution-Pack
# Stub out image segment list block

This adds a place holder implementation of the image segment list block. It calls methods that were stubbed out to provide static responses. The will need to be update to give actual results so that it will have the correct data when it renders.

The block checks if it is on a page with a newspaper page before it displays.

## How to test
Enable the block from Structure -> Blocks, then set the **Image Segment List** block to appear in one of the sidebars.
